src/coreComponents/constitutive/solid/PorousSolid.cpp

```diff
@@ -28,6 +28,7 @@
 #include "DamageSpectral.hpp"
 #include "DamageVolDev.hpp"
 #include "ModifiedCamClay.hpp"
+#include "DuvautLionsSolid.hpp"
 
 namespace geos
 {
```

Comment:
Porous Viscous models, such as `typedef PorousSolid< DuvautLionsSolid< DruckerPrager > > PorousViscoDruckerPrager` , should defined and registered.